deploy/foreman-tasks.sysconfig

```diff
@@ -6,3 +6,15 @@ FOREMAN_LOGGING=warn
 FOREMAN_LOGGING_SQL=warn
 FOREMAN_TASK_PARAMS="-p foreman"
 FOREMAN_LOG_DIR=/var/log/foreman
+
+#Set the number of executors you want to run
+#EXECUTORS_COUNT=1
+
+#Set memory limit for executor process, before it's restarted automatically
+#EXECUTOR_MEMORY_LIMIT=1gb
+
+#Set delay before first memory polling to let executor initialize (in sec)
+#EXECUTOR_INIT_DELAY=180
+
+#Set memory polling interval, process memory will be checked every N seconds.
+#EXECUTOR_POLLING_INTERVAL=60
```

Comment:
@ShimShtein the executor needs to load the whole rails environement, therefore it's really the same as initializing the Katello elsewhere.

@lzap as Shim said, the 180s is guaranteed time that the service will not restart, but doesn't mean it will really restart: it would restart only when it crossed the `EXECUTORS_MEMORY_LIMIT`. I think we could make the default a bit longer, let's say 10 minutes?